chrome/content/zotero/xpcom/data/feed.js

```diff
@@ -89,10 +112,22 @@ Zotero.defineProperty(Zotero.Feed.prototype, 'isFeed', {
 Zotero.defineProperty(Zotero.Feed.prototype, 'libraryTypes', {
 	value: Object.freeze(Zotero.Feed._super.prototype.libraryTypes.concat(['feed']))
 });
+Zotero.defineProperty(Zotero.Feed.prototype, 'unreadCount', {
+	get: function() this._feedUnreadCount
+});
+Zotero.defineProperty(Zotero.Feed.prototype, 'updating', {
+	get: function() this._updating,
+	set: function(v) {
+		if (!v == !this._updating) return; // Unchanged
+		this._updating = !!v;
+		Zotero.Notifier.trigger('statusChanged', 'feed', this.id);
+	}
+});
 
 (function() {
 // Create accessors
-let accessors = ['name', 'url', 'refreshInterval', 'cleanupAfter'];
+let accessors = ['name', 'url', 'refreshInterval', 'cleanupAfter', 
+	'lastCheck', 'lastUpdate', 'lastCheckError'];
```

Comment:
`this._changed` is managed down the line by `Zotero.Item.prototype._set`, required for saving. `this.unreadCount` is a calculated property, not stored in the database. Which I now realise, has problems, because it never gets initialized when a feed is created (restored from DB).